docker/runtests.sh

```diff
@@ -48,15 +48,16 @@ echo -e "\n== Starting memcached"
 sleep 3
 
 echo -e "\n== Updating configuration"
-mysql -u root mysql -e "CREATE DATABASE bugs_test CHARACTER SET = 'utf8';"
+mysql -u root mysql -e "CREATE DATABASE bugs_test CHARACTER SET = 'utf8';" || exit 1
+mysql -u root mysql -e "GRANT ALL PRIVILEGES ON bugs_test.* TO bugs@'%' IDENTIFIED BY 'bugs'; FLUSH PRIVILEGES;" || exit 1
 sed -e "s?%DB%?$BUGS_DB_DRIVER?g" --in-place $BUGZILLA_ROOT/qa/config/checksetup_answers.txt
 sed -e "s?%DB_NAME%?bugs_test?g" --in-place $BUGZILLA_ROOT/qa/config/checksetup_answers.txt
 sed -e "s?%USER%?$BUGZILLA_USER?g" --in-place $BUGZILLA_ROOT/qa/config/checksetup_answers.txt
 echo "\$answer{'memcached_servers'} = 'localhost:11211';" >> $BUGZILLA_ROOT/qa/config/checksetup_answers.txt
 
 if [ "$TEST_SUITE" == "checksetup" ]; then
     cd $BUGZILLA_ROOT/qa
-    /buildbot_step "Checksetup" ./test_checksetup.pl config/config-checksetup-$BUGS_DB_DRIVER
+    /buildbot_step "Checksetup" ./checksetup.pl config/config-checksetup-$BUGS_DB_DRIVER
     exit $?
 fi
```

Comment:
Remove this section completely. This was an old test we no longer do that tests upgrading older databases used by older versions of Bugzilla. so test_checksetup.pl was the correct script to run, not checksetup.pl.